components/extractors/other/telemetry-data/telemetry-data.md

```diff
@@ -0,0 +1,292 @@
+| `kbc_organization_is_deleted` | Flag determining if the organization is deleted (`true`, `false`) | `true` |
+| `kbc_organization_autojoin` | Flag determining if Keboola Support can access projects in the organization without a confirmed request | `true` |
+| `kbc_organization_url` | URL of the organization in KBC | `https://connection.keboola.com/ admin/organizations/225/settings` |
+
+
+### kbc_project
+This table shows data about Keboola Connection projects belonging to an organization.
+
+| **Column** | **Description** | **Example** | 
+| `kbc_project_id` (PK) | KBC project identifier | `1944_us-east-1_aws` |
+| `kbc_project_id_num` | Project numeric identifier | `1944` |
+| `kbc_project` | Name of the project | `Sales Workshop - Kuba` |
+| `kbc_project_region` | Region of the project | `us-east-1` |
+| `kbc_project_cloud` | Cloud provider of the project | `aws` |
+| `kbc_project_url` | URL of the project | `https://connection.keboola.com/ admin/projects/1944` |
+| `kbc_organization_id` | Foreign key to the KBC organization | `116_us-east-1_aws` |
+| `kbc_project_created` | Datetime of when the project was created | `2017-09-04 14:26:40` |
+| `kbc_project_deleted` | Datetime of when the project was deleted | `2017-10-19 14:26:49` |
+| `kbc_project_is_deleted` | Determines if the project is deleted. | `true` |
+| `kbc_project_expiration` | Datetime of the project expiration date | `2017-10-25 10:13:57` |
+| `kbc_project_type` | Type of the project (no actual impact on billing). Possible values: <br> `demo`, `poc`, `poc15Days`, `poc15DaysGuideMode`, `poc30Days`,  `poc6months`, `production`, `payAsYouGo` | `poc` |
+| `kbc_project_creator` | User that created the project | `martin.matejka@keboola.com` |
+
+
+### kbc_project_snapshot
+This table shows snapshots of the projects to track their size changes.
+
+| **Column** | **Description** | **Example** | 
+| `kbc_project_id` (PK) | Foreign key to the KBC project | `1105_eu-central-1_aws` |
+| `snapshot_date` (PK) | Date of the data snapshot | `2020-06-26` |
+| `rows` | Number of rows in the project at the time of the snapshot | `28137568882` |
+| `bytes` | Data size of the project in bytes at the time of the snapshot | `1071491549696` |
+
+
+### kbc_project_user
+This table shows snapshots capturing the projects' admins. The snapshots are taken multiple times a day.
+
+| **Column** | **Description** | **Example** | 
+| `kbc_project_id` (PK) | Foreign key to the KBC project | `409_eu-west-1_azure` |
+| `snapshot_time` (PK) | Datetime of the snapshot | `2020-06-29 12:55:16` |
+| `user_id` (PK) | Identifier of the user | `2195` |
+| `valid_from` | Datetime from which the user is a member of the project | `2020-03-28 11:19:47` |
+| `email` | Email of the user | `martin.matejka@keboola.com` |
+| `domain` | Email domain of the user | `keboola.com` |
+
+
+### kbc_snowflake_stats
+This table shows information about Snowflake usage.
```

Comment:
tyká se to workspaců https://help.keboola.com/transformations/workspace/ nebo i storage jobů https://help.keboola.com/storage/jobs/ ? (anebo to počítá i dotazy komponent?)

Reply:
Obsahuje to queries (agregace) transformací, sandboxů a dotazů do námi poskytnutého schema (u writeru). Storage to neřeší.

Reply:
budou tam tímpádem i dotazy jiných komponent, co používají snowflake workspace (jako snowflake transformace)

Reply:
Možná ti obsah nejlépe shrnou zdrojové tabulky v Padákově transformaci: https://connection.keboola.com/admin/projects/258/transformations/bucket/329722040/transformation/329722090
Nicméně to prostě není jedna část dokumentace, takže moc nevím, co linkovat.

Reply:
Já to až tak vědět nepotřebuju, ale ty lidi, co to budou číst jo :) 
Jestli to nefiltruješ podle komponent, tak tam budou i jiný komponenty - tj. nemůžem říct, že to jsou "pouze transformace a sandboxy", ale že to jsou "**např.** transformace a sandboxy" nebo tak něco.

Reply:
Co takhle?

This table shows information about queries using Snowflake, including transformations and sandboxes.